src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/PrintPackagesResult.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.CommandLine.XPlat.Utility
+{
+    /// <summary>
+    /// Provides meta information found when printing packages.
+    /// </summary>
+    internal readonly struct PrintPackagesResult
+    {
+        public PrintPackagesResult(bool autoReferenceFound, bool deprecatedFound)
+        {
+            AutoReferenceFound = autoReferenceFound;
+            DeprecatedFound = deprecatedFound;
+        }
+
+        /// <summary>
+        /// <c>True</c> when an auto-referenced package was found; otherwise <c>False</c>.
+        /// </summary>
+        public bool AutoReferenceFound { get; }
+
+        /// <summary>
+        /// <c>True</c> when a deprecated package was found; otherwise <c>False</c>.
+        /// </summary>
+        public bool DeprecatedFound { get; }
+    }
+}
```

Comment:
super minor: end with new line.  You didn't rebase to the latest dev? :)

Reply:
I merged dev in several times, but as stated earlier, `CTRL+K+D` shortcut doesn't always seem to add it...

Reply:
Fixed in https://github.com/NuGet/NuGet.Client/pull/2928/commits/f14ddc33891ac566e32eb2f1ce48b766a3385216